documentation/Corpus-documentation.md

```diff
@@ -0,0 +1,19 @@
+# Corpus documentation pages
+
+Corpora can include markdown templates containing documentation. I-analyzer supports several types of documentation pages.
+
+- general information
+- citation guidelines
+- documentation on word models
+- a licence for re-using the data
+- terms of service (usually pro)
+
+All are optional.
+
+### Writing documentation
+
+Documentation files are written in markdown. For information about supported flavours, see [marked.js documentation](https://marked.js.org/) - the frontend uses that package with default options.
```

Comment:
Maybe refer to an existing documentation page in the source code for inspiration.